Rocket.Chat/Controllers/Auth/SignupTableViewController.swift

```diff
@@ -173,7 +173,9 @@ final class SignupTableViewController: BaseTableViewController {
 
         startLoading()
         AuthManager.signup(with: name, email, password, customFields: customFields) { [weak self] response in
-            stopLoading()
+            DispatchQueue.main.async {
```

Comment:
Why do we need to force the main thread here? It should already be in the main thread 🤔

Reply:
Debugger is reporting it's being called in a background thread.